### ISSUE TITLE
libretro-install.sh not copying .so files

### ISSUE THREAD

**Issue by parkerlreed** [libretro/libretro-super#222]
I removed the cores I didn't want from fetch.

Compiled all the cores with libretro-build.sh

Then when I run sudo ./libretro-install.sh (or to a directory I own) it only copies the .info files. 

At the top of the output I get
````
Library /home/parker/build/libretro-super/dist//info not found, skipping...
Library /home/parker/build/libretro-super/dist//unix not found, skipping...
````

Both of these folders exist.

**Comment by hokata** [comment]
I had the same problem yesterday.
I provided a fix in #223

**Comment by hokata** [comment]
This one could been closed. (I forgot to specify "fix ..." in my commit)
